packages/documentation/src/pages/architecture/site-structure/client-overview.mdx

```diff
@@ -0,0 +1,100 @@
+## VA.gov Client Overview
```

Comment:
most of the documents linked in the ticket didn't really relate to site structure. I combined three into this page.